doc/building/building_on_osx.rst

```diff
@@ -1,14 +1,10 @@
-This section contains information about building Tribler on macOS. The final result you should have is a ``.dmg`` file which, when opened, allows ``Tribler.app`` to be copied to the Applications directory. This guide has been tested on macOS 10.11 (El Capitan). It is recommended to run this builder on a system that is already able to run Tribler without problems (it means that all the required packages required by Tribler are installed already). Information about setting up a developer environment on macOS can be found `here <(https://github.com/Tribler/tribler/blob/devel/doc/development/development_on_osx.rst>`_.
+Building Tribler on macOS (10.11). The final result is a ``.dmg`` file which, when opened, allows ``Tribler.app`` to be copied to the Applications directory and or launched. Make sure the required packages required by Tribler are installed 
+from the  `Development instructions <../development/development_on_osx.rst>`_.
 
 Required packages
------------------
-
-To build and distribute Tribler, there are some required scripts and packages:
-* The git command tools are required to fetch the latest release information. They are installed when you start Xcode for the first time but you can also install it using ``brew`` or another package library.
-* PyInstaller: this library creates an executable binary and can be installed using pip (``pip install pyinstaller``).
```

Comment:
As the commit message stated they are duplicates because the "Development instructions" are a requirement for building so git and PyInstaller have already been installed.